File: Instructor Python/Collections/Python-List/Python/ArrayInt.py
import random


class ArrayInt(object):
    @staticmethod
    def rand(size, low, high):
        array = []
        for _ in range(0, size):
            array.append(random.random(low, high))
        return array

    @staticmethod
    def print(array):
        index = 0
        for item in array:
            print(f" index={index:3d} ==>  Value ={item:3d} ")
            index += 1
        return array

File: Instructor Java/collections/Java-Array/ArrayInt.java
import java.util.Random;

/**
 * ArrayDemo
 */
public class ArrayInt {

  public static int[] rand(int size, int low, int high) {
    var ret = new int[size];
    if (low > high) {
      var c = low;
      low = high;
      high = c;
    }
    var bound = high - low;
    var rand = new Random();
    for (var i = 0; i < size; i++) {
      ret[i] = low + rand.nextInt(bound + 1);
    }
    return ret;
  }

  public static void print(int[] arr) {
    var index = 0;
    for (var item : arr) {
      System.out.printf("index=%3d ==>  Value=%3d %n", index++, item);
    }
  }

}
